Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
+
+### Promoting diversity and representation
+
+One of the goals of the OBF fellowship is to increase participation from groups otherwise underrepresented in the community or at the event, including but not limited to underrepresented demographic groups (country of residence and citizenship), ethnic background (historically underrepresented and other minority groups), career stages, gender identity and expression, and people with disabilities.
+
+In an open text response, applicants can describe how their participation in this event help increase diversity and representation of any marginalised and traditionally underrepresented group in the OVERALL open source bioinformatics and/or open science community.
+
+Note that applicants may have an answer to this question even if they are not personally a member of an underrepresented group. 
+This section is important for the evaluation of their application.
+
+### Justification for funding
+
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation related costs of up to USD $1,000. 
+  - For the in-person events, this fellowship can be used to cover airfare, transportation and hotel. 
+  - For the virtual events, applicants can seek support for small hardware, internet cost for the duration of the event, or other supporting materials
+
+Applicants are asked to provide details of the items they would like to pay for through the OBF Event fellowship. 
+Here, they can provide as much detail as possible to help the review panel make a fair recommendation for the fellowship awardees.
+
+Please see the next section for more details.
+
+## What does the event fellowship cover?
+
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation related costs of up to USD $1,000. 
+
+### Items covered for in-person participation
+
+The fellowship can be used to cover conference registration fees and direct travel costs (such as airfare, train, metro, bus and taxi), hotel, and/or child-care up to a value of USD 1,000. 
+Note that rental car expenses and personal car mileage are not eligible for reimbursement. 
+If you anticipate that you would require more than USD 1,000 in order to attend the event, please include that information on the application form and contact the fellowship chair.
+
+### Items covered for remote participation
+
+The fellowship can be used to cover conference registration fees and the costs of small hardware (such as Microphone, speaker and webcam), childcare for the duration of the event and/or other supporting materials.
+
+The cost of remote participation will be evaluated on a case-by-case basis.
+For example, we are aware that due to the load-shedding issues in many developing countries, our applicants might require to pay for the fuel for the generator to keep their electricity running for the duration of the event.
```

Comment:
require to pay -> need to pay. (for some reason, the verb "require" doesn't work before another verb; it is used before nouns, like "require payment" (which is not quite the right phrase here))